shell/src/main/java/tachyon/shell/command/CountCommand.java

```diff
@@ -91,4 +91,9 @@ public void run(String... args) throws IOException {
   public String getUsage() {
     return "count <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Displays the number of files and directories matching the specified prefix.";
```

Comment:
"matching the specified prefix" is confusing to me, I may suspect that "/a/b/c*" will match "/a/b/c", "/a/b/cd" like unix glob. 

What about "Displays the total bytes of a file, or the total bytes, number of files and number of directories recursively contained under a directory."?